libs/hdf-converters/src/gosec-mapper.ts

```diff
@@ -92,9 +108,10 @@ export class GoSecMapper extends BaseConverter {
         return {
           auxiliary_data: [
             {
-              name: 'Gosec',
+              name: 'gosec',
```

Comment:
Why the name change?  And why does it not match the other changes?

Reply:
See above for name change. Passthrough `name` fields also typically only involve just the tool name.